dot-studio/golang

```diff
@@ -202,10 +201,10 @@ function dep_ensure() {
   local go_cmd="dep ensure -vendor-only"
   ( cd "$scaffolding_go_pkg_path" || exit
     # Run dep ensure only if vendor/ directory is missing
-    if [ ! -d vendor ]; then
+    if [[ ! -d vendor ]]; then
       install_go_tool github.com/golang/dep/cmd/dep
       echo "=> Executing Go dependency solver ($go_cmd)"
-      eval $go_cmd
+      eval "$go_cmd"
```

Comment:
In this case I know these `eval`s *are* necessary, but it's because of some deep weirdness. It would be great if someone had time to unpack it and make this code less gnarly, but I don't right now.